bears/c_languages/AStyleLintBear.py

```diff
@@ -0,0 +1,62 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='astyle',
+        output_format='corrected',
+        use_stdin=True)
+class AStyleLintBear:
+    """
+    Artistic Style is a source code indenter, formatter,
+    and beautifier for the C, C++, C++/CLI, Objective‑C,
+    C# and Java programming languages.
+
+    For more information, consult <http://astyle.sourceforge.net/astyle.html>.
+    """
+
+    LANGUAGES = {'C', 'C++', 'Objective-C', 'C#', 'Java'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='astyle')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         astyle_style: str='',
+                         use_spaces: bool=None,
+                         astyle_config: str='',
+                         indent_size: int=0):
+        """
+        :param astyle_style:
+            Defines the bracket style to use. If no bracket style is
+            specified, the default 'allman' is used.
+        :param use_spaces:
+            If None, no indentation is specified and astyle uses a default.
+            If True, astyle uses spaces or if explicitly set to False, astyle
+            uses tabs for indentation.
+        :param astyle_config:
+            Optional configuration file for astyle to read configs.
```

Comment:
a lot of of bears still specify the the linter specific config file. It gets me confused. For example https://github.com/coala/coala-bears/blob/master/bears/python/PyLintBear.py#L38

Not sure though but I think the coala config file is called using `--config-file` or `--config` which is different from the option astyle uses to call it's configuration file. So I don't understand how the two relate.